geocode/cpo.py

```diff
@@ -67,6 +67,8 @@ def _load(self, force_reload : bool = False):
                     cpo = pd.concat([cpo, data]) if cpo is not None else data
         cpo["Postcode"] = cpo["Postcode"].str.replace(" ", "", regex=False)
         cpo["Postcode"] = cpo["Postcode"].str.upper()
+        # remove all rows with 0 Eastings or Northings
+        cpo = cpo[~((cpo["Eastings"] == 0) | (cpo["Northings"] == 0))]
```

Comment:
```suggestion
        cpo = cpo[~((cpo["Eastings"] == 0) & (cpo["Northings"] == 0))]
```

Probably an AND condition is safer here, although I don't think there are any legit (i.e. onshore) E/N coords with 0